### DISCUSSION TITLE
Support OIDC for logging in?

### DISCUSSION THREAD

**Discussion by nogweii** [vinceanalytics/vince#29] [General]
Right now, vince requires it's own username/password pair. I would love to reduce the number of passwords I have to manage by having it instead rely on OpenID Connect to delegate authentication to another service. (In my case, Authentik.)

I think it'd be a great selling point compared to other self-hostable analytics tools as none of them support OIDC.

**Comment by gernest** [comment]
This can be solved by deploying vince behind a reverse proxy. All major reverse proxies support OIDC.

I intend to keep things simple for vince.
